lm_eval/__main__.py

```diff
@@ -180,12 +180,12 @@ def cli_evaluate(args: Union[argparse.Namespace, None] = None) -> None:
         include_path(args.include_path)
 
     if args.tasks is None:
-        task_names = ALL_TASKS
+        eval_logger.error("Need to specify task to evaluate.")
+        sys.exit()
     elif args.tasks == "list":
         eval_logger.info(
-            f"Available Tasks:\n - {(os.linesep + ' - ').join(sorted(ALL_TASKS))}"
+            "Available Tasks:\n - {}".format("\n - ".join(task_manager.all_tasks()))
         )
-        sys.exit()
```

Comment:
when ran `lm_eval --tasks list`
fails with
`UnboundLocalError: local variable 'task_names' referenced before assignment`
without this exit

Reply:
Thanks, will make a fix.